### ISSUE TITLE
Exported image margin

### ISSUE THREAD

**Issue by shawky** [literallycanvas/literallycanvas#282]
Is there is a way to add margin around the exported image?

**Comment by irskep** [comment]
`lc.getImage({rect: {x, y, width, height}})` lets you specify any bounds you want.

**Comment by shawky** [comment]
I tried the method you mentioned, it sets the width and height of the exported image as well the origin of the image but I couldn't add the required margin.

Seems I am missing something or I didn't explain myself sufficiently.

I was pointing to, when drawing something on the canvas then exporting it, the exported image shows the drawings just to the edge of the image without a margin around it. My question is how to add a margin around those drawings.

I am attaching two images to better explain the case, the first is the exported image without margins, while on the second image margins was added around the drawing (I filled it with yellow color).

Thank you
![index](https://cloud.githubusercontent.com/assets/3769250/8430914/ebe73806-1f3d-11e5-99c0-eca03e29d818.png)
![index2](https://cloud.githubusercontent.com/assets/3769250/8430917/f11763e6-1f3d-11e5-8e2f-7a4aab05ac2d.png)
